cli/src/plugins/javadoc/index.ts

```diff
@@ -420,17 +420,19 @@ const makeClassDocPageBody: MakeBodyFunction = (args) => {
       makeBody: () =>
         makeTable(
           ["Constructor and Description"],
-          doc.constructors.map((doc) => [
-            [
-              md.paragraph([
-                md.text("|   "),
-                project.linkToEntity(doc.qualifiedName, doc.name),
-                md.text(" "),
-                ...makeParameterListWithLinks(project, doc),
-              ]),
-              md.paragraph(tagsToMdast(project, doc.firstSentenceTags)),
-            ],
-          ])
+          doc.constructors
+            .filter((doc) => !doc.modifiers.startsWith("protected"))
+            .map((doc) => [
+              [
+                md.paragraph([
+                  md.text("|   "),
```

Comment:
Is this rST syntax?

Reply:
Yep, this seems to be the only way to reliably force tabbing without triggering a snooty error about block quotes or causing Snooty to bold the first line because it assumes that the entire paragraph is a "definition". We're deep in the Dark Arts of not-really-rst-spec with this one, in standard rST there is a better way but that doesn't work in our platform.

Reply:
Ok, well it doesn't belong as a `text` then as if the "yokedast" were rendered to something other than rST, the semantics of this would be lost. Instead, why not make a new yokedast element that represents whatever you're trying to do, then add a yokedast -> rst renderer specifically for this.

Reply:
This just fixes a small bug with the vertical bar implementation in one place (it's used in a couple of places), so I would suggest that we should add a ticket to create a separate component in the yokedast to the Trello board and then migrate each instance over once we have that component. But I'm not sure it's something we need to treat as a blocker for this bugfix.

Reply:
I'm not sure it's worth taking on technical debt for this when we could just do it properly now...

Reply:
Fair enough, I'll create a component for this instead.